go_client_README.md

```diff
@@ -25,6 +25,8 @@ func NewBlueMatadorClient() *BlueMatadorClient
 
 ```
 `NewBlueMatadorClient` instantiates a BlueMatadorClient with the default host ("localhost") and port (8767) address. If you have set `BLUEMATADOR_AGENT_HOST` and `BLUEMATADOR_AGENT_PORT` in the config file for your agent these variables will be used for the address. 
```

Comment:
I thought these actually came from the environment variables. That's how the other clients work. So they're not read from any file. They're just environment variables.

Reply:
I think it should be

`NewBlueMatadorClient` instantiates a BlueMatadorClient with the default host ("localhost") and port (8767) address. If you have set `BLUEMATADOR_AGENT_HOST` and `BLUEMATADOR_AGENT_PORT` as env vars, they will be used for the address.